snuba/consumers/snapshot_worker.py

```diff
@@ -85,7 +85,10 @@ def __drop_message(self, xid: int) -> Optional[ProcessedMessage]:
             )
         return None
 
-    def process_message(
+    # Typing does not allow a subclass to have a more specific argument
+    # since it violates the Liskov substitution principle.
+    # https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides
+    def process_message(  # type: ignore
```

Comment:
Sorry, the typing of MessageProcessor.process_message was incorrect earlier. I updated it to Any in https://github.com/getsentry/snuba/pull/1643.

Reply:
Oh sweet OK I'll revert this.